packages/rsk3-utils/src/index.js

```diff
@@ -777,7 +783,7 @@ const padLeft = (string, chars, sign) => {
  * @returns {String}
  */
 const toTwosComplement = (number) => {
-    if (!isNumber(number)) {
+    if (!isNumber(number) && !isString(number) && !isBN(number)) {
         throw new Error('number parameter should be a number.');
```

Comment:
You have changed the checks, and then you need to change the output(error information as well)